### PR TITLE
W-17403599-AsyncAPI-to-MuleSoft-VCS-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@
  ** xref:flow-counting-studio.adoc[Track the Number of Billable Flows in Studio]
  ** xref:test-specification-api-console.adoc[Test your API Specification Using the API Console]
 
-* xref:api-sync.adoc[API Sync]
+* xref:api-sync.adoc[Sync Your Mule Projects with MuleSoft VCS]
```
Comment (simoneegeib) [inline modules/ROOT/nav.adoc]: this only affects API projects, not Mule projects. I would change this to "Sync your API projects with MuleSoft VCS"

Comment (Crispy-Salesforce) [reply modules/ROOT/nav.adoc]: OK! Thanks!

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -25,7 +25,7 @@
  ** xref:flow-counting-studio.adoc[Track the Number of Billable Flows in Studio]
  ** xref:test-specification-api-console.adoc[Test your API Specification Using the API Console]
 
-* xref:api-sync.adoc[API Sync]
+* xref:api-sync.adoc[Sync Your API Projects with MuleSoft VCS]
```
Comment (Crispy-Salesforce) [inline modules/ROOT/nav.adoc]:
```suggestion
* xref:api-sync.adoc[Sync Your API Projects with API Sync]
```

File: modules/ROOT/pages/api-sync.adoc [hunk 3]
```diff
@@ -1,71 +1,58 @@
-= API Sync
+= Sync Your API Projects with MuleSoft VCS
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
= Sync Your API Projects with API Sync
```

File: modules/ROOT/pages/api-sync.adoc [hunk 4]
```diff
@@ -1,71 +1,58 @@
...
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: api, anypoint platform, sync, api sync
+:keywords: api, anypoint platform, sync, api sync, MuleSoft VCS
 
-The API Sync feature of Anypoint Studio enables you to:
+With MuleSoft VCS, you can:
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
With API Sync, you can:
```

File: modules/ROOT/pages/api-sync.adoc [hunk 5]
```diff
@@ -1,71 +1,58 @@
...
 
-* Start designing a RAML, or OAS 2.0 and 3.0 API specification in Studio and sync it with MuleSoft VCS.
-* Pull a published RAML or OAS 2.0 and 3.0 API specification from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. +
-API sync also enables you to edit the API specification you imported from Exchange, and push your changes to MuleSoft VCS.
-+
-You can use API Sync on more than one API specification to scaffold multiple flows in different Mule configuration files.
-* Monitor the API versions that you implemented in your project and update your flows based on the latest published version.
-* Pull a RAML or OAS 2.0 and 3.0 API specification from MuleSoft VCS into Studio, edit the specification offline, and pull the updates back to MuleSoft VCS.
+* Start designing a RAML, or OAS 2.0 and 3.0 API spec in Studio and sync it with MuleSoft VCS.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
* Start designing a RAML, or OAS 2.0 and 3.0 API spec in Studio and sync it with API Sync.
```

File: modules/ROOT/pages/api-sync.adoc [hunk 6]
```diff
@@ -1,71 +1,58 @@
...
+* Pull a published RAML or OAS 2.0 and 3.0 API spec from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. 
+* You can edit the API spec you imported from Exchange, and push your changes to MuleSoft VCS.
+* You can use MuleSoft VCS on more than one API spec to scaffold multiple flows in different Mule configuration files.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
* You can use API Sync on more than one API spec to scaffold multiple flows in different Mule configuration files.
```

File: modules/ROOT/pages/api-sync.adoc [hunk 7]
```diff
@@ -1,71 +1,58 @@
...
+* Monitor the API versions that you implement in your project and update your flows based on the latest published version.
+* Pull a RAML or OAS 2.0 and 3.0 API spec from MuleSoft VCS into Studio, edit the spec offline, and pull the updates back to MuleSoft VCS.
+* Develop your Mule applications following API Lifecycle development practices.
 
-API Sync supports importing both RAML, OAS 2.0 and OAS 3.0 specifications.
+MuleSoft VCS supports importing both RAML, OAS 2.0, and OAS 3.0 spec, comes bundled by default with Studio, and requires Mule 4.1.4 and later.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
API Sync supports importing both RAML, OAS 2.0, and OAS 3.0 spec, comes bundled by default with Studio, and requires Mule 4.1.4 and later.
```

File: modules/ROOT/pages/api-sync.adoc [hunk 8]
```diff
@@ -1,71 +1,58 @@
...
 
-API Sync comes bundled by default in Studio, and requires Mule 4.1.4 and later.
+MuleSoft VCS can’t work properly if the RAML file exceeds 4,000 lines of text.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
API Sync can’t work properly if the RAML file exceeds 4,000 lines of text.
```

File: modules/ROOT/pages/api-sync.adoc [hunk 9]
```diff
@@ -1,71 +1,58 @@
...
+See xref:sync-api-projects-design-center.adoc[] for more information.
 
-=== Edit The Imported API Specification in Studio
+=== Edit the Imported API Spec in Studio
 
-You can also edit the imported API specification in your project as an API specification project.
+You can also edit the imported API spec in your project as an API spec project.
 
-. Import the API specification from Exchange to your Studio project.
+. Import the API spec from Exchange to your Studio project.
```
Comment (simoneegeib) [inline modules/ROOT/pages/api-sync.adoc:41]: I think this is a bit confusing. This eems to be about importing a spec from Exchange. Why do you import it, change it, and then import it again?

File: modules/ROOT/pages/api-sync.adoc [hunk 10]
```diff
@@ -1,71 +1,58 @@
...
+== Create an API Spec in Studio
 
-Using API Sync, you can develop your Mule applications following API Lifecycle development practices.
+. Start a new API spec project in Studio.
++
+See xref:create-api-specification-studio.adoc[] for more information.
+. Publish it to Exchange.
++
+See xref:publish-api-project-to-exchange.adoc[]
+. Import the same API spec from Exchange to your Studio project.
```
Comment (simoneegeib) [inline modules/ROOT/pages/api-sync.adoc]: You should leave out teh last part of importing teh API Spec into Studio. It makes no sense as you already created the API in Studio and exported to Exchange

File: modules/ROOT/pages/api-sync.adoc (hunk 9 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc:41]: According to my understanding, you do this process to have two versions of the same API spec.

File: modules/ROOT/pages/api-sync.adoc (hunk 10 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: The purpose of this doc is to explain how the API Sync feature works. 
If you think this is redundant, I can delete the last part.

**Review by simoneegeib — DISMISSED** [review]

File: modules/ROOT/nav.adoc [hunk 11]
```diff
@@ -25,7 +25,7 @@
  ** xref:flow-counting-studio.adoc[Track the Number of Billable Flows in Studio]
  ** xref:test-specification-api-console.adoc[Test your API Specification Using the API Console]
 
-* xref:api-sync.adoc[API Sync]
+* xref:api-sync.adoc[Sync Your API Projects with API Sync]
```
Comment (hannanelson) [inline modules/ROOT/nav.adoc]: Why the hard-coded link text?

File: modules/ROOT/pages/api-sync.adoc [hunk 12]
```diff
@@ -1,71 +1,50 @@
-= API Sync
+= Sync Your API Projects with API Sync
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: api, anypoint platform, sync, api sync
+:keywords: api, anypoint platform, sync, api sync, MuleSoft VCS
```
Comment (hannanelson) [inline modules/ROOT/pages/api-sync.adoc:5]: Is API Sync the new name for MuleSoft VCS?

File: modules/ROOT/pages/api-sync.adoc [hunk 13]
```diff
@@ -1,71 +1,50 @@
...
+With API Sync, you can:
 
-* Start designing a RAML, or OAS 2.0 and 3.0 API specification in Studio and sync it with MuleSoft VCS.
-* Pull a published RAML or OAS 2.0 and 3.0 API specification from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. +
-API sync also enables you to edit the API specification you imported from Exchange, and push your changes to MuleSoft VCS.
-+
-You can use API Sync on more than one API specification to scaffold multiple flows in different Mule configuration files.
-* Monitor the API versions that you implemented in your project and update your flows based on the latest published version.
-* Pull a RAML or OAS 2.0 and 3.0 API specification from MuleSoft VCS into Studio, edit the specification offline, and pull the updates back to MuleSoft VCS.
+* Start designing a RAML, or OAS 2.0 and 3.0 API spec in Studio and sync it with API Sync.
```
Comment (hannanelson) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
* Start designing a RAML, OAS 2.0, or OAS 3.0 API spec in Studio and sync it with API Sync.
```

File: modules/ROOT/pages/api-sync.adoc [hunk 14]
```diff
@@ -1,71 +1,50 @@
...
+* Pull a published RAML or OAS 2.0 and 3.0 API spec from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project.
```
Comment (hannanelson) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
* Pull a published RAML, OAS 2.0, or OAS 3.0 API spec from Exchange or from a private Maven repository, into your Mule application project in Studio to scaffold a new project. 
```

File: modules/ROOT/pages/api-sync.adoc (hunk 12 above)
Comment (hannanelson) [reply modules/ROOT/pages/api-sync.adoc:5]: Oh, nm. I guess not.

File: modules/ROOT/pages/api-sync.adoc [hunk 15]
```diff
@@ -1,71 +1,50 @@
...
-* Pull a published RAML or OAS 2.0 and 3.0 API specification from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. +
-API sync also enables you to edit the API specification you imported from Exchange, and push your changes to MuleSoft VCS.
-+
-You can use API Sync on more than one API specification to scaffold multiple flows in different Mule configuration files.
-* Monitor the API versions that you implemented in your project and update your flows based on the latest published version.
-* Pull a RAML or OAS 2.0 and 3.0 API specification from MuleSoft VCS into Studio, edit the specification offline, and pull the updates back to MuleSoft VCS.
+* Start designing a RAML, or OAS 2.0 and 3.0 API spec in Studio and sync it with API Sync.
+* Pull a published RAML or OAS 2.0 and 3.0 API spec from Exchange or a private Maven repository, into your Mule application project in Studio to scaffold a new project. 
+* Edit the API spec you imported from Exchange, and push your changes to MuleSoft VCS.
+* Use API Sync on more than one API spec to scaffold multiple flows in different Mule configuration files.
```
Comment (hannanelson) [inline modules/ROOT/pages/api-sync.adoc]: Flip around? Scaffold multiple flows by using....

File: modules/ROOT/pages/api-sync.adoc [hunk 16]
```diff
@@ -1,71 +1,50 @@
...
+* Monitor the API versions that you implement in your project and update your flows based on the latest published version.
+* Pull a RAML or OAS 2.0 and 3.0 API spec from MuleSoft VCS into Studio, edit the spec offline, and pull the updates back to MuleSoft VCS.
```
Comment (hannanelson) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
* Pull a RAML, OAS 2.0, or OAS 3.0 API spec from MuleSoft VCS into Studio, edit the spec offline, and push the updates back to MuleSoft VCS.
```

File: modules/ROOT/pages/api-sync.adoc [hunk 17]
```diff
@@ -1,71 +1,50 @@
...
+* Develop your Mule applications following API Lifecycle development practices.
```
Comment (hannanelson) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
* Develop your Mule applications using API lifecycle development practices.
```

File: modules/ROOT/pages/api-sync.adoc [hunk 18]
```diff
@@ -1,71 +1,50 @@
...
 
-API Sync supports importing both RAML, OAS 2.0 and OAS 3.0 specifications.
+API Sync supports importing both RAML, OAS 2.0, and OAS 3.0 spec, comes bundled by default with Studio, and requires Mule 4.1.4 and later.
```
Comment (hannanelson) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
API Sync supports importing RAML, OAS 2.0, and OAS 3.0 specs, is bundled by default with Studio, and requires Mule 4.1.4 and later.
```

Comment (hannanelson) [reply modules/ROOT/pages/api-sync.adoc]: Do you need the first bit? Isn't it obvious from the previous bullets? 

API Sync is bundled by default with Studio and requires Mule 4.1.4 or later.

File: modules/ROOT/pages/api-sync.adoc [hunk 19]
```diff
@@ -1,71 +1,50 @@
...
 
-API Sync comes bundled by default in Studio, and requires Mule 4.1.4 and later.
+API Sync can’t work properly if the RAML file exceeds 4,000 lines of text.
```
Comment (hannanelson) [inline modules/ROOT/pages/api-sync.adoc]:
```suggestion
API Sync doesn't work properly if the RAML file exceeds 4,000 lines of text.
```

File: modules/ROOT/pages/api-sync.adoc [hunk 20]
```diff
@@ -1,71 +1,50 @@
...
 
-== Create an API Specification in Studio
+== Import an API Spec from Exchange 
 
-. Start a new API specification project in Studio.
-See xref:create-api-specification-studio.adoc[Create an API Specification Project in Studio] for more information.
-. Publish it to Exchange.
-. Import the same API specification from Exchange to your Studio project. +
-See xref:import-api-specification-exchange.adoc[Import an API Specification from Exchange] for more information.
+Import an API spec from Exchange into a new Mule project or from MuleSoft VCS into an Editable API Spec Project.
```
Comment (hannanelson) [inline modules/ROOT/pages/api-sync.adoc]: Can this be lowercase: Editable API Spec Project

File: modules/ROOT/pages/api-sync.adoc (hunk 12 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc:5]: No. API Sync is the features Studio has to connect with MuleSoft VCS or other repos.

File: modules/ROOT/pages/api-sync.adoc (hunk 13 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: OK! Thanks!

File: modules/ROOT/pages/api-sync.adoc (hunk 14 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: OK! Thanks!

File: modules/ROOT/pages/api-sync.adoc (hunk 16 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: OK! Thanks!

File: modules/ROOT/pages/api-sync.adoc (hunk 17 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: OK! Thanks!

File: modules/ROOT/pages/api-sync.adoc (hunk 19 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: OK! Thanks!

File: modules/ROOT/nav.adoc (hunk 11 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/nav.adoc]: You're right! 
Thanks!

File: modules/ROOT/pages/api-sync.adoc (hunk 15 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: Great idea! Thanks!

File: modules/ROOT/pages/api-sync.adoc (hunk 18 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: Importing and pulling may be obvious to someone that understands the function of a source control system.
As this is the introduction of API Sync, it may be somehow useful to mention that API Sync can import and pull specs. Thanks!

File: modules/ROOT/pages/api-sync.adoc (hunk 20 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: Yes! Great catch! Thanks!

File: modules/ROOT/pages/api-sync.adoc (hunk 18 above)
Comment (hannanelson) [reply modules/ROOT/pages/api-sync.adoc]: OK, then suggest rewording to separate the supports from the other bits. As it is, it's kind of a mishmash of thoughts.

bundled and requires
supports

**Review by hannanelson — APPROVED** [review]

File: modules/ROOT/pages/api-sync.adoc (hunk 18 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/api-sync.adoc]: I'll create a WI to work on that! 
Thanks!
